pages.es/common/huggingface-cli.md

```diff
@@ -0,0 +1,37 @@
+# huggingface-cli
+
+> Interactúa con Hugging Face Hub.
+> Inicia sesión, gestiona la caché local, carga o descarga archivos.
+> Más información: <https://huggingface.co/docs/huggingface_hub/guides/cli>.
+
+- Inicia sesión en Hugging Face Hub:
+
+`huggingface-cli login`
+
+- Muestra el nombre del usuario conectado:
+
+`huggingface-cli whoami`
+
+- Cierra sesión:
+
+`huggingface-cli logout`
+
+- Genera información sobre el entorno:
+
+`huggingface-cli env`
+
+- Descarga archivos de un repositorio e imprime la ruta (omite los nombres de archivo para descargar todo el repositorio):
+
+`huggingface-cli download --repo-type {{tipo_de_repositorio}} {{id_de_repositorio}} {{nombre_archivo1 nombre_archivo2 ...}}`
+
+- Sube una carpeta entera o un archivo a Hugging Face:
+
+`huggingface-cli upload --repo-type {{tipo_de_repositorio}} {{id_de_repositorio}} {{ruta/a/archivo_de_repositorio_o_directorio_de_repositorio}}`
```

Comment:
In the English page, there are 4 placeholders.

Reply:
fixed